numba/np/arraymath.py

```diff
@@ -771,6 +786,50 @@ def array_argmax_impl_generic(arry):
     return max_idx
 
 
+def build_argmax_or_argmin_with_axis_impl(arr, axis, flatten_impl):
```

Comment:
This function's code is unchanged, it's simply moved to the top-level (whereas previously it was internal to `array_argmax`) so it can be used by both argmin and argmax implementations.

Reply:
Confirmed. Thanks.